src/client.rs

```diff
@@ -1245,6 +1246,37 @@ impl<T: Read + Write + Unpin + fmt::Debug + Send> Session<T> {
         Ok(uids)
     }
 
+    /// The [`GETQUOTA` command](https://tools.ietf.org/html/rfc2087#section-4.2)
+    pub async fn get_quota(&mut self, quota_root: Cow<'_, str>) -> Result<Quota<'_>> {
```

Comment:
`Cow` seems to be an overkill here. I would prefer a simple `quota_root: &str`, because you are going to immediately copy from this string anyway inside of `quote!`. Other methods in this module use `impl AsRef<str>`, so you may use `quota_root: impl AsRef<str>` and then `quote!(quota_root.as_ref())`, but this will lead to duplication of code if the function is used both with `&str` and `String`, see https://github.com/deltachat/deltachat-core-rust/pull/2425.

@dignifiedquire Do you think we need to maintain consistency and use `impl AsRef<str>` or may use simple `&str` and maybe later change other functions such as `login`, `authenticate` etc. to use `&str` and make a major release?